src/Microsoft.AspNetCore.ResponseCompression/project.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "1.1.0-*",
+    "buildOptions": {
+        "warningsAsErrors": true,
+        "keyFile": "../../tools/Key.snk",
+        "nowarn": [
+            "CS1591"
+        ],
+        "xmlDoc": true
+    },
+    "description": "ASP.NET Core basic middleware for HTTP Response compression.",
+    "packOptions": {
+        "repository": {
+            "type": "git",
+            "url": "git://github.com/aspnet/basicmiddleware"
+        },
+        "tags": [
+            "aspnetcore"
+        ]
+    },
+    "dependencies": {
+        "Microsoft.AspNetCore.Http": "1.1.0-*",
```

Comment:
Middleware shouldn't need to reference Http directly. What are you using from here that isn't in abstractions?